### ISSUE TITLE
[REQ] Option to disable closing the app by pressing back button

### ISSUE THREAD

**Issue by Solarunit** [jonjomckay/fritter#504]
**Describe the problem that needs solving**

It would be nice to have an option to disable closing the app by pressing back button.
I often accidentally close the app thinking that I am exiting fullscreen image/video preview.

**Describe the solution you'd like**

I am thinking about 2 options:

1. Option in the setting to disable closing the app by pressing back button.
2. Notification **"Tap again to exit"** at the bottom which saying that the next pressing back button will close the app.

**Comment by geominorai** [comment]
I tried this in the CI build and being unable to back out to exit the app is unexpected behavior not standard for Android apps. 

Can showing the edit confirmation prompt please be a toggle in settings and off by default (off being the previous behavior)?

**Comment by geominorai** [comment]
_Sync for Reddit_ has an less intrusive alternative to a confirmation dialog where it flashes a notice at the bottom to press back a second time to exit the app:

<img src="https://user-images.githubusercontent.com/13911156/192091179-dc3a85af-9e6b-43a7-833a-b2f0b9677c1c.png" width=400 />

**Comment by Solarunit** [comment]
> _Sync for Reddit_ has a less intrusive alternative to a confirmation dialog where it flashes a notice at the bottom to press back a second time to exit the app:

Yep, I mean something like that. I saw similar implementation in another apps.
